packages/material-ui/src/CheckboxesWidget/CheckboxesWidget.tsx

```diff
@@ -76,13 +76,7 @@ const CheckboxesWidget = ({
               onFocus={_onFocus}
             />
           );
-          return inline ? (
```

Comment:
I try not to refactor within unrelated commits but this seemed relevant since the code block removed used the `inline` property. I'm not sure why this was done but this is redundant regardless because both sides of the ternary return the same thing.